packages/core/src/card/card.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Card" />
+
+# Card
+
+## Installation
+
+```typescript
+import '@cds/core/card/register.js';
+```
+
+## Usage
+
+```html
+<cds-card>
+  <div cds-layout="vertical gap:md">
+    <div class="header" cds-layout="p-t:lg p-l:lg">
+      <p cds-text="title">Profile of John Doe</p>
+    </div>
+
+    <cds-divider></cds-divider>
+
+    <div class="body" cds-layout="p:lg">
+      <p cds-text="p1">Profile information</p>
+    </div>
+
+    <cds-divider></cds-divider>
+
+    <div class="footer" cds-layout="p:lg">
+      <cds-button> View </cds-button>
+    </div>
+  </div>
+</cds-card>
+```
+
+<Preview>
+  <Story id="card-stories--basic" />
+</Preview>
+
+## API
+
+<Props of={'cds-card'} />
```

Comment:
Do you know why the slot items show up even though there are no slots for cds-card?

Reply:
Card extends the panel. The panel has a slot element, iirc.